### ISSUE TITLE
get_binary_labels returned non binary labels.

### ISSUE THREAD

**Issue by Salamander321** [CAMMA-public/cholect50#1]
In get_binary_labels function in dataloader (pytorch) i found,
```py
if triplet[0] != -1.0:
                triplet_label[triplet[0]] =+ 1
```
So when i try printing the labels of instrument i found this.
![image](https://user-images.githubusercontent.com/34837784/220373015-4cf94324-1f0e-4822-8305-72a104cd706b.png)
Instead,  of +=, shouldn't it be = like following ?
```py
if triplet[0] != -1.0:
                triplet_label[triplet[0]] = 1
```

**Comment by TheBlackCat22** [comment]
Has this issue been solved?

**Comment by nwoyecid** [comment]
Dear @Salamander321, the code is correct depending on your usage.
For classification task, setting label = 1 is fine, or you can write another code to convert every number > 0 to 1.
For instance localization, setting label += 1 preserves the number of instances.
